client/my-sites/stats/activity-log/index.jsx

```diff
@@ -494,6 +495,7 @@ class ActivityLog extends Component {
 				<SidebarNavigation />
 				<StatsNavigation selectedItem={ 'activity' } siteId={ siteId } slug={ slug } />
 				{ siteId && <ActivityLogUpgradeNotice siteId={ siteId } /> }
+				{ 'unavailable' === rewindState.state && <UnavailabilityNotice siteId={ siteId } /> }
```

Comment:
In the line before we make sure `siteId` is available before passing it as a prop. Maybe we should do the same here.
```js
{ ( siteId && 'unavailable' === rewindState.state ) && <UnavailabilityNotice siteId={ siteId } /> }
```

Reply:
hm. a good point. in this case there would be no bug I think because we couldn't get `unavailable` if there were no `siteId`, but it's probably just as good or better to include the extra check